scripts/api.js

```diff
@@ -0,0 +1,116 @@
+import PocketChange from './pocket-change.js';
+import MacroSupport from './macro-support.js';
+import Settings from './settings.js';
+
+const API = {
```

Comment:
This applies to all `.js` files, but I use prettier and that should probably be ran on all files for formatting.

Reply:
wil do